_rules/object-has-acessible-name-8fc3b6.md

```diff
@@ -38,12 +41,20 @@ The `object` element is not rendered for presentational purposes. If the `object
 
 ## Accessibility Support
 
-Non-supported media formats make screen readers render the text content of the element instead of other attributes.
+There are no major accessibility support issues known for this rule.
 
 ## Background
 
 Testing that the [accessible name][] describes the purpose of the `object` element is not part of this rule and must be tested separately.
 
+Non-supported media formats make screen readers render the text content of the element instead of other attributes. 
+
+`Object` elements without an accessible name are ignored by assistive technologies unless they have an [explicit role][].
+
+When the object resource is not loaded the fallback content is rendered as shown in Inapplicable Example 7.
```

Comment:
Calling out an example by number feels prone to be missed if the example numbering changes. I'm okay with it, but how do we normally make sure these references don't break?